README.md

```diff
@@ -8,12 +8,12 @@
 
 ## Introduction
 
-Platform for AI is a cluster management tool and resource scheduling platform, jointly designed and developed by [Microsoft Research (MSR)](https://www.microsoft.com/en-us/research/group/systems-research-group-asia/) and [Microsoft Search Technology Center (STC)](https://www.microsoft.com/en-us/ard/company/introduction.aspx).
-The platform incorporates some mature design that has a proven track record in large scale Microsoft production environment, and is tailored primarily for academic and research purpose. 
+Platform for AI (PAI) is a platform for cluster management and resource scheduling.
```

Comment:
What is the final official name, PAI or OpenPAI?

Reply:
we can use it interchangeably. 

---
In reply to: [189809895](https://github.com/Microsoft/pai/pull/596#discussion_r189809895) [](ancestors = 189809895)